### PR TITLE
Add container deletion before creation

### PR DESCRIPTION
**Pull request by injectives**
This is just to make sure the build does not fail if there is such container already (possibly from a previous build).

### CODE REVIEW

**Comment by robsdedude** [comment]
A problem I faced myself often enough :+1:

File: docker.py [hunk 1]
```diff
@@ -39,8 +39,10 @@ def rm(self):
         del _running[self.name]
 
 
-def create(image, name, command=None, mount_map=None, host_map=None, port_map=None,
-           env_map=None, working_folder=None, network=None, aliases=None):
+def createOrReplace(image, name, command=None, mount_map=None, host_map=None,
```
Comment (robsdedude) [inline docker.py]: `create_or_replace` would be the pythonic way to go here.

**Review by robsdedude — CHANGES_REQUESTED** [review]

File: docker.py (hunk 1 above)
Comment (injectives) [reply docker.py]: fixed 👍

**Review by robsdedude — APPROVED** [review]
Didn't run it myself, but looks good to me.

**Pull request merged**